_sass/components/partners/_partners.scss

```diff
@@ -21,6 +21,11 @@
     }
 }
 
+.partners-body-wide {
+    @extend .partners-body;
+    max-width: 880px;
+}
+
```

Comment:
Rather than creating this as a separate CSS class which copies all of the styles from the original, I wonder if we could implement this as a [BEM modifier](https://en.bem.info/methodology/css/#modifiers), which overrides only the width property, and then assign `partners-body partners-body--wide` classes to the element.

```suggestion
.partners-body--wide {
    max-width: 880px;
}
```